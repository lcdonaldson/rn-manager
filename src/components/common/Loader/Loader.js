import React, { Component } from 'react';
import { TouchableOpacity, ActivityIndicator} from 'react-native';

class Loader extends React.Component {
    render() {
        return (
            <View>
                <TouchableOpacity>
                    <ActivityIndicator />
                </TouchableOpacity> 
            </View>        
        )
    }
}
