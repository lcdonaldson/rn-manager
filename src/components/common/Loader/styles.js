import {StyleSheet} from 'react-native';

export default StyleSheet.create({

    loaderLayout: {
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
        backgroundColor: '#00000060,',
    },
    cancelBtn: {
        paddingTop: 10,
        paddingBottom: 10,
        paddingLeft: 30,
        paddingRight: 30,
        color: '#fff',
        backgroundColor: "teal",
    }
    
})
