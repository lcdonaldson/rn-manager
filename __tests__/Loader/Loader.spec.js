import React from "react";
import 'react-native';
import render from 'react-test-renderer';
import Loader from '../../src/components/common/Loader/Loader';

describe("Loader component", () => {
  it("snapshot tests should match wrapper", () => {
    const tree = render.create(<Loader/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
})
