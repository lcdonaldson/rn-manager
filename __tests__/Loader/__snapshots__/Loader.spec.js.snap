// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component snapshot tests should match wrapper 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#fff"
    hidesWhenStopped={true}
    size="large"
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "teal",
        "color": "#fff",
        "opacity": 1,
        "paddingBottom": 10,
        "paddingLeft": 30,
        "paddingRight": 30,
        "paddingTop": 10,
      }
    }
  />
</View>
`;
